sycl/source/detail/event_impl.hpp

```diff
@@ -209,6 +209,11 @@ class event_impl {
   }
   bool needsCleanupAfterWait() { return MNeedsCleanupAfterWait; }
 
+  /// Returns worker queue for command.
+  ///
+  /// @return a reference to MWorkerQueue.
+  QueueImplPtr &getWorkerQueue() { return MWorkerQueue; };
```

Comment:
Is it both setter and getter method?
Couldn't find a c-tor which sets worker queue.
```suggestion
  const QueueImplPtr &getWorkerQueue() const { return MWorkerQueue; };
```

Reply:
it has the same approach as with MPreparedTaskDeps. WorkerQueue can be different depending on command type and it is assigned by command in its constructor.